src/data-loader.ts

```
@@ -1335,6 +1371,72 @@ export async function loadBucketUsageData(
+ * Based on the Python reference implementation for better accuracy
+ * @param transcriptPath - Path to the transcript JSONL file
+ * @returns Object with context tokens info or null if unavailable
+ */
+export async function calculateContextTokens(transcriptPath: string): Promise<{
+	inputTokens: number;
+	percentage: number;
+	contextLimit: number;
+} | null> {
+	let content: string;
+	try {
+		content = await readFile(transcriptPath, 'utf-8');
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)